### PR TITLE
SqlRSSetup: Installing SSRS 2019 in integration test

### LINKED ISSUE #1717: SQLRSSetup: Should install Microsoft SQL Server 2019 Reporting Services in integration test
Download URL:
https://download.microsoft.com/download/1/a/a/1aaa9177-3578-4931-b8f3-373b24f63342/SQLServerReportingServices.exe

### PR DESCRIPTION
**Pull request by johlju**
#### Pull Request (PR) description
- SqlRSSetup
  - Integration tests is now installing _Microsoft SQL Server 2019 Reporting Services_
    (issue #1717).

#### This Pull Request (PR) fixes the following issues
- Fixes #1717

#### Task list

- [x] Added an entry to the change log under the Unreleased section of the
      file CHANGELOG.md. Entry should say what was changed and how that
      affects users (if applicable), and reference the issue being resolved
      (if applicable).
- [ ] Resource documentation updated in the resource's README.md.
- [ ] Resource parameter descriptions updated in schema.mof.
- [ ] Comment-based help updated, including parameter descriptions.
- [ ] Localization strings updated.
- [ ] Examples updated.
- [ ] Unit tests updated. See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] Integration tests updated (where possible). See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] Code changes adheres to [DSC Community Style Guidelines](https://dsccommunity.org/styleguidelines).

---
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718)

### CODE REVIEW

File: tests/Integration/README.md [hunk 1]
```diff
@@ -99,12 +102,22 @@ SSRS | Stopped
...
-- **LogPath:** C:\Users\appveyor\AppData\Local\Temp\SSRS
+- **LogPath:** Unknown
+- **InstallFolder:** C:\Program Files\Microsoft SQL Server Reporting Services
+- **ServiceName:** SQLServerReportingServices
+- **Edition:** Developer
+
+### Properties for the instance SSRS 2019
+
+- **InstanceName:** SSRS
+- **CurrentVersion:** ^15.0.1102.911 (depends on the version downloaded)
```
Comment (johlju) [inline tests/Integration/README.md]: Version reported by integration test is 15.0.7842.32355

**Review by PlagueHO — CHANGES_REQUESTED** [review]
> Reviewed 1 of 3 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718)* status: all files reviewed, 3 unresolved discussions (waiting on @johlju)

___
*[CHANGELOG.md, line 56 at r2](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718#-MiIVsll5AYkZ0Tvn1I4:-MiIVsll5AYkZ0Tvn1I5:b-rc035l) ([raw file](https://github.com/dsccommunity/sqlserverdsc/blob/ea04e994a68f327250815eb43a6b0fed3492260e/CHANGELOG.md#L56)):*
> ```
> - Integration tests now install _Microsoft SQL Server 2019 Reporting Services_
> ```
___
*[tests/Integration/DSC_SqlRSSetup.Integration.Tests.ps1, line 3 at r2](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718#-MiIW4B05UcFq5p_nFM-:-MiIW4B05UcFq5p_nFM0:b-2kffsh) ([raw file](https://github.com/dsccommunity/sqlserverdsc/blob/ea04e994a68f327250815eb43a6b0fed3492260e/tests/Integration/DSC_SqlRSSetup.Integration.Tests.ps1#L3)):*
> ```PowerShell
> Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath '..\TestHelpers\CommonTestHelper.psm1')
> 
> # Run only for version of Microsoft SQL Server Reporting Services.
> ```

Do you mean this runs only for any version of Microsoft SQL Server Reporting Services? I wasn't 100% clear on this. Looks like it runs for two specific versions.

**Review by johlju — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718)* status: 1 of 3 files reviewed, 3 unresolved discussions (waiting on @PlagueHO)

___
*[CHANGELOG.md, line 56 at r2](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718#-MiIVsll5AYkZ0Tvn1I4:-MiKcLkA6MF4fTttPem2:b-896fix) ([raw file](https://github.com/dsccommunity/sqlserverdsc/blob/ea04e994a68f327250815eb43a6b0fed3492260e/CHANGELOG.md#L56)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary><blockquote>

> ```
> - Integration tests now install _Microsoft SQL Server 2019 Reporting Services_
> ```

</blockquote></details>

Done.
___
*[tests/Integration/DSC_SqlRSSetup.Integration.Tests.ps1, line 3 at r2](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718#-MiIW4B05UcFq5p_nFM-:-MiKcOCvBTTO64n-oHhG:boytghe) ([raw file](https://github.com/dsccommunity/sqlserverdsc/blob/ea04e994a68f327250815eb43a6b0fed3492260e/tests/Integration/DSC_SqlRSSetup.Integration.Tests.ps1#L3)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary><blockquote>

Do you mean this runs only for any version of Microsoft SQL Server Reporting Services? I wasn't 100% clear on this. Looks like it runs for two specific versions.
</blockquote></details>

Done. Reading it back again it was not grammatically correct or even said what I meant. :laughing:  Clarified it, now I hope it is better. :slightly_smiling_face:

**Review by johlju — COMMENTED** [review]
:lgtm:

I merging this so I can rebase the other PR. 

> *[Reviewable](https://reviewable.io/reviews/dsccommunity/sqlserverdsc/1718#-:-MiQpGYABdV6rjMhWL1t:bthucpf)* status: 1 of 3 files reviewed, 3 unresolved discussions (waiting on @PlagueHO)

**Pull request merged**